src/GLX/libglx.c

```diff
@@ -1975,6 +1971,48 @@ static void __glXResetOnFork(void)
     __glXMappingTeardown(True);
 }
 
+PUBLIC const __glXGLCoreFunctions __GLXGL_CORE_FUNCTIONS = {
+    glXChooseFBConfig,
+    glXChooseVisual,
+    glXCopyContext,
+    glXCreateContext,
+    glXCreateGLXPixmap,
+    glXCreateNewContext,
+    glXCreatePbuffer,
+    glXCreatePixmap,
+    glXCreateWindow,
+    glXDestroyContext,
+    glXDestroyGLXPixmap,
+    glXDestroyPbuffer,
+    glXDestroyPixmap,
+    glXDestroyWindow,
+    glXGetClientString,
+    glXGetConfig,
+    glXGetCurrentContext,
+    glXGetCurrentDrawable,
+    glXGetCurrentReadDrawable,
+    glXGetFBConfigAttrib,
+    glXGetFBConfigs,
+    glXGetProcAddress,
+    glXGetProcAddressARB,
+    glXGetSelectedEvent,
+    glXGetVisualFromFBConfig,
+    glXIsDirect,
+    glXMakeContextCurrent,
+    glXMakeCurrent,
+    glXQueryContext,
+    glXQueryDrawable,
+    glXQueryExtension,
+    glXQueryExtensionsString,
+    glXQueryServerString,
+    glXQueryVersion,
+    glXSelectEvent,
+    glXSwapBuffers,
+    glXUseXFont,
+    glXWaitGL,
+    glXWaitX,
+};
+
```

Comment:
We already have this list in _LIBGLX_FUNCTIONS. Duplicating it here seems unnecessary.